tools/crm_mon.c

```diff
@@ -101,6 +101,68 @@ static pcmk__supported_format_t formats[] = {
     { NULL, NULL, NULL }
 };
 
+PCMK__OUTPUT_ARGS("crm-mon-disconnected", "const char *", "const char *",
+                  "int");
+static int
+crm_mon_disconnected_default(pcmk__output_t *out, va_list args)
+{
+    const char *msg = va_arg(args, const char *);
+    const char *description = va_arg(args, const char *);
+    bool reset_and_finish = (bool) va_arg(args, int);
+    int rc = pcmk_rc_ok;
+
+    msg = pcmk__s(msg, "Cluster is not available");
+
+    if (reset_and_finish) {
+        out->reset(out);
+    }
+
+    if (description != NULL) {
+        rc = out->info(out, "%s (%s)", msg, description);
+    } else {
+        rc = out->info(out, "%s", msg);
+    }
+
+    if (reset_and_finish) {
+        out->finish(out, CRM_EX_DISCONNECT, true, NULL);
+    }
+    return rc;
+}
+
+PCMK__OUTPUT_ARGS("crm-mon-disconnected", "const char *", "const char *",
+                  "int");
+static int
+crm_mon_disconnected_xml(pcmk__output_t *out, va_list args)
+{
+    const char *msg = va_arg(args, const char *);
+    const char *description = va_arg(args, const char *);
+    bool reset_and_finish = (bool) va_arg(args, int);
+
+    msg = pcmk__s(msg, "Cluster is not available");
+
+    if (reset_and_finish) {
+        out->reset(out);
+    }
+
+    if (description != NULL) {
+        out->err(out, "%s (%s)", msg, description);
+    } else {
+        out->err(out, "%s", msg);
+    }
```

Comment:
I'm thinking we don't always want this as an error -- aren't we now calling this before the first connection?

Reply:
Almost certainly better to use the pacemakerd-health message. It's already on my to-do list to do that via `pcmk__pacemakerd_status()`. Wouldn't hurt to do it via pacemakerd-health now, given that we improved that since this PR was first filed.